src/App.css

```diff
@@ -193,118 +472,239 @@
+  background: var(--task-bg);
+  border-radius: 4px;
+  padding: 16px;
+  border: 1px solid var(--border-color);
+  color: var(--text-primary);
 }
 
-.color-picker-container {
-  position: absolute;
-  z-index: 1000;
-  background: white;
-  border-radius: 8px;
-  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.15);
-  padding: 8px;
-  margin-top: 8px;
-  right: 16px;
+.task-metadata {
+  border-top: 1px solid var(--border-color);
+  padding-top: 16px;
+  margin-top: 24px;
 }
 
-.color-options {
-  display: grid;
-  grid-template-columns: repeat(4, 1fr);
+.metadata-item {
+  display: flex;
   gap: 8px;
+  margin-bottom: 8px;
+  color: var(--text-secondary);
+  font-size: 14px;
 }
 
-.color-option {
-  width: 24px;
-  height: 24px;
-  border-radius: 50%;
-  border: none;
-  cursor: pointer;
-  transition: transform 0.2s ease;
+.metadata-label {
+  font-weight: 500;
+  color: var(--text-primary);
 }
 
-.color-option:hover {
-  transform: scale(1.1);
+/* Markdown Editor Styles */
+.w-md-editor {
+  background: var(--task-bg) !important;
+  border: 1px solid var(--border-color) !important;
+  border-radius: 4px !important;
+  color: var(--text-primary) !important;
 }
 
-.color-picker-button {
-  width: 20px;
-  height: 20px;
-  border-radius: 50%;
-  border: 2px solid white;
+.w-md-editor-toolbar {
+  background: var(--column-background) !important;
+  border-bottom: 1px solid var(--border-color) !important;
+}
+
+.w-md-editor-toolbar li > button {
+  color: var(--text-secondary) !important;
+}
+
+.w-md-editor-toolbar li.active > button,
+.w-md-editor-toolbar li > button:hover {
+  color: var(--text-primary) !important;
+  background: var(--hover-bg) !important;
+}
+
+.w-md-editor-text {
+  background: var(--task-bg) !important;
+  color: var(--text-primary) !important;
+}
+
+.w-md-editor-text-input {
+  color: var(--text-primary) !important;
+  background: var(--task-bg) !important;
+}
+
+.w-md-editor-text-pre > code {
+  color: var(--text-primary) !important;
+  background: var(--task-bg) !important;
+}
+
+.w-md-editor-preview {
+  background: var(--task-bg) !important;
+  box-shadow: inset 1px 0 0 0 var(--border-color) !important;
+}
+
+.wmde-markdown {
+  background: transparent !important;
+  color: var(--text-primary) !important;
+}
+
+/* Ensure text visibility in both themes */
+[data-color-mode="dark"] .w-md-editor-text-input {
+  -webkit-text-fill-color: var(--text-primary) !important;
+}
+
+[data-color-mode="light"] .w-md-editor-text-input {
+  -webkit-text-fill-color: var(--text-primary) !important;
+} 
+
+/* Task Detail Header Actions */
+.task-detail-actions {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.delete-task-button {
+  padding: 6px 12px;
+  background: transparent;
+  border: 1px solid var(--border-color);
+  color: var(--text-secondary);
+  border-radius: 4px;
   cursor: pointer;
-  box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1);
-  transition: transform 0.2s ease;
+  font-size: 14px;
+  transition: all 0.2s ease;
 }
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
.delete-task-button {
  padding: 6px 12px;
  background: transparent;
}

.delete-task-button:disabled {
  opacity: 0.6;
  cursor: not-allowed;
}
```
Add a disabled state for the delete button to prevent multiple clicks during deletion process



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;